src/main/scala/io/iohk/ethereum/jsonrpc/EthJsonMethodsImplicits.scala

```diff
@@ -174,6 +175,60 @@ object EthJsonMethodsImplicits extends JsonMethodsImplicits {
         value = value.getOrElse(0),
         data = data.getOrElse(ByteString("")))
     }
+  }
+
+  implicit val eth_getCode = new JsonDecoder[GetCodeRequest] with JsonEncoder[GetCodeResponse] {
+    def decodeJson(params: Option[JArray]): Either[JsonRpcError, GetCodeRequest] =
+      params match {
+        case Some(JArray((address: JString) :: (blockStr: JString) :: Nil)) =>
+          for {
+            addr <- tryExtractUnformattedData(address)
+            block <- tryExtractBlockParam(blockStr)
+          } yield GetCodeRequest(addr, block)
+        case _ => Left(InvalidParams())
+      }
+
+    def encodeJson(t: GetCodeResponse): JValue = encodeAsHex(t.result)
+  }
+
+  implicit val eth_getUncleCountByBlockNumber = new JsonDecoder[GetUncleCountByBlockNumberRequest] with JsonEncoder[GetUncleCountByBlockNumberResponse] {
+    def decodeJson(params: Option[JArray]): Either[JsonRpcError, GetUncleCountByBlockNumberRequest] =
+      params match {
+        case Some(JArray((blockStr: JString) :: Nil)) =>
+          for {
+            block <- tryExtractBlockParam(blockStr)
+          } yield GetUncleCountByBlockNumberRequest(block)
+        case _ => Left(InvalidParams())
+      }
+
+    def encodeJson(t: GetUncleCountByBlockNumberResponse): JValue = encodeAsHex(t.result)
+  }
+
+  implicit val eth_getUncleCountByBlockHash = new JsonDecoder[GetUncleCountByBlockHashRequest] with JsonEncoder[GetUncleCountByBlockHashResponse] {
+    def decodeJson(params: Option[JArray]): Either[JsonRpcError, GetUncleCountByBlockHashRequest] =
+      params match {
+        case Some(JArray((blockHashStr: JString) :: Nil)) =>
+          for {
+            blockHash <- tryExtractUnformattedData(blockHashStr)
```

Comment:
Minor detail: As with `eth_getCode` should we also check here that the `blockHashStr` has the correct length and return an error message instead of `null`?